cmd/ltops/destroy.go

```diff
@@ -23,8 +25,11 @@ var destroyCluster = &cobra.Command{
 		if err != nil {
 			return errors.Wrap(err, "Couldn't load cluster")
 		}
-
-		return cluster.Destroy()
+		mlog.Info("Destroying cluster...")
+		if err = cluster.Destroy(); err == nil {
```

Comment:
Can we switch this around to be idiomatic?

Reply:
@crspeller - do you mean turn it into 'err != nil' ?

Reply:
Yes. Exactly.